### DISCUSSION TITLE
how to skip parameter not show in allure report steps @Step()

### DISCUSSION THREAD

**Discussion by BBSoftPro** [allure-framework/allure2#2099] [Questions & Support] [answered]
```
@Step()
void confirmPasscode(int seconds, passcode){
enterPasscode(passcode)
delay(second)
}
```
in this case if I want to skip seconds cos I don't want to show it in the report steps. have any command this annotation to skip the showing parameter in report?

**Comment by baev** [comment, marked as answer]
You can use `io.qameta.allure.Param` annotation for that:



```java
@Step
void confirmPasscode(@Param(mode=HIDDEN) int seconds, passcode){
    // ...
}
```

**Reply by KozyrevOleg** [reply]
@baev good evening!
Looks like there is a problem with the @Param tag. I found an error for the "exclude" option. When I removed this setting manually from the *result.json file, the same error started appearing for "mode".

```
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "mode" (class io.qameta.allure.model.Parameter), not marked as ignorable (2 known properties: "value", "name"])
 at [Source: (sun.nio.ch.ChannelInputStream); line: 1, column: 9501] (through reference chain: io.qameta.allure.model.TestResult["steps"]->java.util.ArrayList[3]->io.qameta.allure.model.StepResult["parameters"]->java.util.ArrayList[0]->io.qameta.allure.model.Parameter["mode"])
```

or maybe I should change my dependencies?

<testing.version>7.6.1</testing.version>
<allure-testng.version>2.24.0</allure-testng.version>
<allure-maven.version>2.12.0</allure-maven.version>


<groupId>com.fasterxml.jackson.core</groupId>
<artifactId>jackson-annotations</artifactId>
<version>2.15.2</version>
<scope>compile</scope>

**Reply by baev** [reply]
You need to specify the latest `allure-commandline` version for `allure-maven` plugin
